config/sitemap.rb

```diff
@@ -1,8 +1,8 @@
 # Set the host name for URL creation
-SitemapGenerator::Sitemap.default_host = "https://ir.library.oregonstate.edu"
+SitemapGenerator::Sitemap.default_host = ENV.fetch('SCHOLARSARCHIVE_URL_RAILS', 'https://ir.library.oregonstate.edu')
 SitemapGenerator::Sitemap.create_index = true
 SitemapGenerator::Sitemap.compress = :all_but_first
-SitemapGenerator::Sitemap.public_path = '/data0/hydra/shared/public/'
+SitemapGenerator::Sitemap.public_path = '/data/tmp/public/'
```

Comment:
Is this production safe?

Reply:
no, added a SCHOLARSARCHIVE_PATH_PUBLIC env key, with the path to the "public" directory. made the default be the production path, and I'll add that key to the production local_env.yml